### PR TITLE
Disable ArangoChair and subscriptions

### PR DESCRIPTION
**Pull request by alex-novikov-1990**

### CODE REVIEW

File: src/server/data/collection.ts [hunk 1]
```diff
@@ -243,6 +243,7 @@ export class QDataCollection {
                 request: QRequestContext,
                 info: { operation: { selectionSet: SelectionSetNode } },
             ) => {
+                throw new Error("Disabled");
```
Comment (melsomino) [inline src/server/data/collection.ts:246]: We need to fix subscriptions on QDataCombiner for hot and accounts databases

**Review by melsomino — CHANGES_REQUESTED** [review]
See comments

File: src/server/data/arango-provider.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ export class ArangoProvider implements QDataProvider {
     async start(collectionsForSubscribe: string[]): Promise<void> {
         this.started = true;
         this.collectionsForSubscribe = collectionsForSubscribe;
-        this.checkStartListener();
+        // this.checkStartListener();
```
[the remark this answers is not in the document]
Comment (alex-novikov-1990) [reply src/server/data/arango-provider.ts:62]: Done

File: src/server/data/arango-provider.ts [hunk 3]
```diff
@@ -101,6 +101,7 @@ export class ArangoProvider implements QDataProvider {
     }
 
     subscribe(collection: string, listener: (doc: QDoc, event: QDataEvent) => void): unknown {
+        throw new Error("Disabled");
```
[the remark this answers is not in the document]
Comment (alex-novikov-1990) [reply src/server/data/arango-provider.ts:104]: Done

**Pull request closed without merging**